core/src/smartcontracts/isi/asset.rs

```diff
@@ -16,6 +16,50 @@ use super::prelude::*;
 pub mod isi {
     use super::*;
 
+    #[allow(clippy::expect_used, clippy::unwrap_in_result)]
+    impl Execute for Register<Asset> {
```

Comment:
`Register<Asset>` and `Unregister<Asset>` should be placed in `smartcontracts/account.rs` since that seems to be the convention

Reply:
Right, I guess I messed up while rebasing and it ended back there.